swap_meet/vendor.py

```diff
@@ -1,2 +1,101 @@
+from swap_meet.item import Item
+
+# Wave 1
+"""The class vendor takes in a parameter of inventory. It has a method function
+    that adds and removes items.
+    Returns:
+      It returns the item or the boolean false.
+        """
+
+
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+        self.item = Item()
```

Comment:
We should **NOT** use mutable types like lists as a default parameter. We should avoid doing this at all costs as it will make all Vendor instances share the exact same list. For example:

If we create two Vendors that will use the fault inventory value:
```
veronica = Vendor()
audrey = Vendor()
```
This would mean veronica and audrey would share the same empty list. Every time veronica adds an item to their inventory, the same item would appear in Audrey's (since it's the same list!) We do not want instances to share the same list, especially if they'll be swapping items in the future. 

To prevent this from happening, we can assign inventory to a default value of "None" if no inventory list is provided. Then when we assign the instance variable `self.inventory`, we can use a ternary to determine the value of `inventory`. 

```suggestion
    def __init__(self, inventory=None):
        self.inventory = inventory if inventory is not None else []
```

Reply:
This article uses functions as examples, but the same rule still applies for constructors (since constructors are also functions).

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/